### PR TITLE
Issue/rviz capsule

### PR DESCRIPTION
**Pull request by Levi-Armstrong**

### CODE REVIEW

File: tesseract_ros/tesseract_rviz/src/render_tools/link_widget.cpp [hunk 1]
```diff
@@ -868,31 +868,44 @@ bool LinkWidget::createEntityForGeometryElement(const tesseract_scene_graph::Lin
...
+                            static_cast<float>(cone.getLength()));
+      break;
+    }
+    case tesseract_geometry::GeometryType::CAPSULE:
+    {
+      const tesseract_geometry::Capsule& capsule = static_cast<const tesseract_geometry::Capsule&>(geom);
+      entity = scene_manager_->createEntity(entity_name, "tesseract_capsule.mesh");
+      scale = Ogre::Vector3(static_cast<float>(capsule.getRadius() * 2),
+                            static_cast<float>(capsule.getRadius() * 2),
+                            static_cast<float>((0.5 * capsule.getLength()) + capsule.getRadius()));
```
Comment (mpowelson) [inline tesseract_ros/tesseract_rviz/src/render_tools/link_widget.cpp]: I assume these scalings are correct? If not, that could be hard to debug later.

File: tesseract/tesseract_collision/test/collision_box_capsule_unit.cpp [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+
+  CollisionShapesConst obj2_shapes;
+  tesseract_common::VectorIsometry3d obj2_poses;
+  obj2_shapes.push_back(thin_box);
+  obj2_poses.push_back(thin_box_pose);
+
+  checker.addCollisionObject("thin_box_link", 0, obj2_shapes, obj2_poses, false);
+
+  /////////////////////////////////////////////////////////////////
+  // Add capsule to checker. If use_convex_mesh = true then this
```
Comment (mpowelson) [inline tesseract/tesseract_collision/test/collision_box_capsule_unit.cpp]: Should this be tested with use_convex_mesh = true as well? I don't see that anywhere.

Comment (Levi-Armstrong) [reply tesseract/tesseract_collision/test/collision_box_capsule_unit.cpp]: I will remove the comment

File: tesseract_ros/tesseract_rviz/src/render_tools/link_widget.cpp (hunk 1 above)
Comment (Levi-Armstrong) [reply tesseract_ros/tesseract_rviz/src/render_tools/link_widget.cpp]: Yea I verified in RViz

**Pull request merged**